inlong-manager/manager-web/bin/restart.sh

```diff
@@ -36,10 +36,7 @@ BIN_PATH=$(
 echo "restart in" ${BIN_PATH}
 
 # Stop service
-sh "$BIN_PATH"/shutdown.sh
```

Comment:
Why change this script here?

Reply:
![image](https://user-images.githubusercontent.com/18047329/222657602-eef876b1-b637-432d-ac77-9d084ab88902.png)

Reply:
@fuweng11 Please revert this file's change, thanks.